<launch>

	<arg name="veh"/>
	<arg name="cont_mode" default="false" doc="tells image transformer to listen to continouus topic,.."/>
	<arg name="trafo_mode" default="lin" doc="'cb' for colo balance only; 'lin' for linear trafo only; 'both' for both"/>
	<param name="~veh" value="$(arg veh)"/>
	<param name="~cont_mode" type="boolean" value="$(arg cont_mode)"/>
	<param name="~trafo_mode" type="str" value="$(arg trafo_mode)"/>

	<group ns="$(arg veh)">


	    <node name="image_transformer_node" pkg="anti_instagram" type="image_transformer_node.py" output="screen">
            <!--remap from="/tesla/camera_node/image/compressed" to="/$(arg veh)/image_transformer_node/uncorrected_image" /-->
        </node>


    </group>

</launch>
